server/src/main/java/com/datastrato/gravitino/server/web/rest/GroupOperations.java

```diff
@@ -54,10 +58,19 @@ public Response getGroup(
     try {
       return Utils.doAs(
           httpRequest,
-          () ->
-              Utils.ok(
-                  new GroupResponse(
-                      DTOConverters.toDTO(accessControlManager.getGroup(metalake, group)))));
+          () -> {
+            NameIdentifier identifier = NameIdentifier.ofMetalake(metalake);
+            return TreeLockUtils.doWithTreeLock(
+                identifier,
+                LockType.READ,
+                () -> {
+                  AuthorizationUtils.checkIsMetalakeOwner(identifier);
```

Comment:
Why does the creator only has the permission to do this operation? What if the owner is not available any more?

Reply:
I think you should have a different role for metalake admins to have the metalake operations.

Reply:
By referring other systems, you should have a series of role and permission system for admin users, like create metalake, access metalake, create/drop role/user/groups. Otherwise, your current solution has lots of limitations.